msedeno23/pemexlogisticaapp
# Feature/modify session management and home UI

**Description**:
This Pull Request modifies the session management to no longer display the hamburger menu for logout. The session will be maintained until the user logs out, is inactive for more than 1 hour, or has no ongoing trip. Additionally, it updates the home interface to be similar to the provided image and adheres to SOLID principles.

**Changes**:
- Removed the hamburger menu for logging out.
- Updated session management to maintain the session until the user logs out, is inactive for 1 hour, or has no ongoing trip.
- Redesigned the home interface to match the provided image.
- Ensured that new screens and modifications adhere to SOLID principles.
- Updated documentation to reflect the changes in session management and home UI.

**Tests**:
1. Run the app locally.
2. Verify that the hamburger menu for logout is no longer displayed.
3. Test the session management to ensure it maintains the session as specified.
4. Navigate to the home screen and ensure it matches the provided design.
5. Ensure all new screens and modifications follow SOLID principles.
